### DISCUSSION TITLE
Advanced Multi-Display Support for CAD-Workstations

### DISCUSSION THREAD

**Discussion by phlahr** [rustdesk/rustdesk#9678] [Q&A]
Hi Guys - first of all nice Work with RustDesk 😉👍

To My Setup:
Serverside: Headless Workstation + 3 Dummy Dongles each with FullHD 60fps support and Nividia Quadro P2000 with Win11 23H2
Clientside: Laptop with Docking (itec) - 3 Monitors each FullHD + one for communication with Intel Iris and Win11 23H2
RustDesk Nightly 1.3.2 - direct Lan Connection
Codec: AV1 with YUV 4:4:4, 100% bitrate an 60fps (yes HWC is enabled)

Im currently trying to setup Multi Monitor with some issues:
1. I can setup all three Monitors in Fullscreen but when i close the connection and try reconnect i always need to rearrange the different monitors of the host again - any advices? (I also tried to edit the .toml (%appdata%\RustDesk\config\peers) of the peer i want to connect but it just shows width, height and x,y-offset of one screen)
2. Is it possible to close the connection for alle 3 monitors if i click on disconnect in one window (like Splashtop)? At the moment i need to close each window individually.
Maybe the Implementation of a "WideScreen"-Mode like in Z-Central (RGS previously), where the windows spans the whole 3 monitors in fullscreen-mode would be great (advantage: you would also only have one toolbar instead of three)
I know i can set the windows to show me all monitors and then stretch it, but only in windowed mode.
4. Im also running some CAD (Allplan) on my Workstation - tested it with following Setups:
- setup in the Header
- both Server an client with dedicated graphics Nvidia quadro
... but both weren´t able to reach more than 26fps wich feels kinda laggy in comparison with 60fps directly on the machine.
AV1 is a highly advanced and compressed and therefore slower than H.265(HEVC) but i need a useful color (YUV 4:4:4 is only possible with AV1) without artefacts.
Is there any way to get more fps?

Thank you Guys in advance for ur answers 🙋‍♂️

**Comment by basilgello** [comment]
Try accelerated VP8 - it may reach hundred fps. But the color issue…

**Reply by phlahr** [reply]
Thx Mate.
Just tried VP8 - reached nearly 40fps but you where right - the color issue ...
What u mean how difficult whould it be to implement a newer codec like LCEVC (Low Complexity Enhancement Video Coding) or some other wich supports YUV 4:4:4?

**Reply by basilgello** [reply]
I have been working on HW playback acceleration on Android and the toughest here is OGL part. Codec itself is doable if there is a library that implements it either in pure Rust or C or C++. But again, I am not employed by comoany behind Rustdesk :(
